### DISCUSSION TITLE
SDXL?

### DISCUSSION THREAD

**Discussion by NeusZimmer** [Amblyopius/Stable-Diffusion-ONNX-FP16#38] [General]
Hello, i would like to ask if there will be any option to convert SDXL model to ONNX with fp16?, the script within optimum only works with CUDA cards...(O4)
Or maybe someone could share a FP16 version of the model already converted into ONNX?...

**Comment by Amblyopius** [comment]
Hi, I plan to have a look soon. VRAM may be an issue but with some loading/unloading I'm sure it'll work eventually. If I can get it going I'll add it to the repo (possibly with a temporary UI).

**Comment by phineas-pta** [comment]
there's the official onnx here: https://huggingface.co/stabilityai/stable-diffusion-xl-1.0-tensorrt

(it's "tensorrt" in the title but the files are onnx)

**Comment by NeusZimmer** [comment]
I will look into it, but each model that i saw was full32 instead of 16, that increases a lot the memory consumption of the model
Anyway , i will try to adapt my UI to the SDXL pipeline soon, and check about it...currently adapting it to the optimum pipelines (txt2img and hires txt2img ready & working... but for SD)...
